tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/SparkTestBase.scala

```diff
@@ -45,20 +45,16 @@ trait SparkTestBase extends AnyFunSuite
     shutdownMiniCluster()
   }
 
-  def updateSparkConf(sparkConf: SparkConf, sort: Boolean): SparkConf = {
+  def updateSparkConf(sparkConf: SparkConf, shuffleWriteMode: String): SparkConf = {
```

Comment:
How about using `ShuffleMode` enum for args type?

Reply:
Thanks, changed